test/bin/pyutils/build_bootc_images.py

```diff
@@ -180,10 +200,20 @@ def extract_container_images(version, repo_spec, outfile, dry_run=False):
     common.popd()
 
 
+def run_gomplate(ifile, ofile, dry_run, logfile):
+    gomplate_args = [
+        GOMPLATE,
+        "--file", ifile,
+        "--out", ofile
+    ]
+    common.run_command_in_shell(gomplate_args, dry_run, logfile, logfile)
+
+
 def process_containerfile(groupdir, containerfile, dry_run):
     cf_path = os.path.join(groupdir, containerfile)
     cf_outname = os.path.splitext(containerfile)[0]
     cf_outdir = os.path.join(BOOTC_IMAGE_DIR, cf_outname)
+    cf_gomplate = os.path.join(BOOTC_IMAGE_DIR, f"{containerfile}.gomplate")
```

Comment:
Interpretation/readability nit: i would expect it to be other way round: `Containerfile.tpl` -> `Containerfile`

Reply:
Changed this to keep the same file name as we're not using `.tpl` in the source currently.